src/App.tsx

```diff
@@ -8,11 +9,26 @@ export const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+  const [selectedTitle, setSelectedTitle] = useState(tabs[0].title);
+
+  return (
+    <div className="App">
+      <div className="container">
+        <h1 className="title is-3">
+          Selected tab is&nbsp;
+          {selectedTitle}
+        </h1>
+        <Tabs
+          tabs={tabs}
+          selectedTabId={selectedTabId}
+          onTabSelected={setSelectedTabId}
+          onTitleSelected={setSelectedTitle}
```

Comment:
You should implement extra functions for onTabSelected and onTitleSelected, because passing the state  isn't a good practice